src/shared/localShadowRepo.ts

```diff
@@ -280,7 +280,7 @@ export class ShadowRepo {
           });
         } catch (e) {
           if (e instanceof git.Errors.MultipleGitError) {
-            this.logger.error('multiple errors on git.add', e.errors.slice(0, 5));
+            this.logger.error(`${e.errors.length} errors on git.add, showing the first 5:`, e.errors.slice(0, 5));
```

Comment:
any specific reason it shows only the first 5 errors?

Reply:
large deployments could result a 4-5 digit number of these?